### PR TITLE
Init Storybook for our frontend components

### PR DESCRIPTION
**Pull request by AbdilFattahBoshi**
HIII @NourSammour  I have Init Storybook for our frontend components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,18 +8,22 @@
...
-
   },
   "dependencies": {
-    "express": "^4.14.0",
+    "express": "^4.17.1",
     "next": "^9.0.2",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "server": "^1.0.19"
```
Comment (NourSammour) [inline package.json]: why this `"server": "^1.0.19"` ?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,14839 @@
+{
```
Comment (NourSammour) [inline package-lock.json]: we are using yarn - not npm - so please install dependencies by using yarn

**Review by NourSammour — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (AbdilFattahBoshi) [reply package.json]: As I told you I had some errors and couldn't test my work so I have tried some of solutions 
now I have remove it

File: package-lock.json (hunk 2 above)
Comment (AbdilFattahBoshi) [reply package-lock.json]: Ok I'll do and for now I have removed package-lock.json file

**Review by NourSammour — APPROVED** [review]

**Pull request merged**